classes/migration/upgrade/v3_5_0/I9678_RemoveScheduledTasksTable.php

```diff
@@ -1,30 +1,40 @@
 <?php
 
 /**
- * @file classes/migration/install/ScheduledTasksMigration.php
+ * @file classes/migration/upgrade/v3_5_0/I9678_RemoveScheduledTasksTable.php
  *
- * Copyright (c) 2014-2021 Simon Fraser University
- * Copyright (c) 2000-2021 John Willinsky
+ * Copyright (c) 2024 Simon Fraser University
+ * Copyright (c) 2024 John Willinsky
  * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
  *
- * @class ScheduledTasksMigration
+ * @class I9678_RemoveScheduledTasksTable
  *
- * @brief Describe database table structures.
+ * @brief Remove scheduled_tasks table.
+ * @see https://github.com/pkp/pkp-lib/issues/9678
+ * 
  */
 
-namespace PKP\migration\install;
+namespace PKP\migration\upgrade\v3_5_0;
 
+use PKP\migration\Migration;
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Support\Facades\Schema;
 
-class ScheduledTasksMigration extends \PKP\migration\Migration
+class I9678_RemoveScheduledTasksTable extends Migration
 {
     /**
-     * Run the migrations.
+     * Run the migration.
      */
     public function up(): void
     {
-        // The last run times of all scheduled tasks.
+        Schema::drop('scheduled_tasks');
```

Comment:
There are other places to update:

https://github.com/pkp/medra/blob/28a5fc1f63469005306715323a19b5a373bc8ad7/MedraDoiDataMigration.php#L312

https://github.com/pkp/pln/blob/226737802d31cff7870ee6b803cdd41288c588e1/classes/migration/install/SchemaMigration.php#L65

Reply:
I plan to update all those but waiting for the implementation to finalise first .